Source/Meadow.Units/Azimuth.cs

```diff
@@ -95,9 +99,16 @@ public Azimuth16PointCardinalNames Compass16PointCardinalName
         /// <summary>
         /// Creates a new `Azimuth` object from a unit value in _Decimal Degrees_ (`°`).
         /// </summary>
-        /// <param name="value">The cardinal direction value.</param>
+        /// <param name="degrees">The cardinal direction value.</param>
+        /// <returns>A new cardinal direction object.</returns>
+        [Pure] public static Azimuth FromDecimalDegrees(double degrees) => new(ConvertTo360(degrees));
+
+        /// <summary>
+        /// Creates a new `Azimuth` object from a unit value in Radians.
+        /// </summary>
+        /// <param name="radians">The cardinal direction value.</param>
         /// <returns>A new cardinal direction object.</returns>
-        [Pure] public static Azimuth FromDecimalDegrees(double value) => new(ConvertTo360(value));
```

Comment:
Any reason to remove FromDecimalDegrees? seems like a reasonable helper method

Reply:
It's still there at line 104